pkg/setting/setting_unified_alerting.go

```diff
@@ -79,6 +79,14 @@ type UnifiedAlertingSettings struct {
 	HARedisPassword                string
 	HARedisDB                      int
 	HARedisMaxConns                int
+	HARedisTLSEnabled              bool
+	HARedisTLSCertPath             string
+	HARedisTLSKeyPath              string
+	HARedisTLSCAPath               string
+	HARedisTLSServerName           string
+	HARedisTLSInsecureSkipVerify   bool
+	HARedisTLSCipherSuites         string
+	HARedisTLSMinVersion           string
```

Comment:
[nit] We could use `dstls.ClientConfig` here, would save an instance of re-listing the options ([here](https://github.com/grafana/grafana/pull/87567/files#diff-1119cd9032e891b1af5337e4d7962436ed5f28914f61fb0f4393c1f845fe4ac7R181-R189))